partner_account_multicompany_default/views/partner_view.xml

```diff
@@ -82,4 +82,14 @@
         </field>
     </record>
 
+    <record id="view_account_position_form" model="ir.ui.view">
+        <field name="name">account.fiscal.position.form.inherit</field>
+        <field name="model">account.fiscal.position</field>
+        <field name="inherit_id" ref="account.view_account_position_form" />
```

Comment:
ask : tree view and search view not need to be update with the code ?

Reply:
I think it's fine to show only in the form. But it can be an improvement indeed